include/yaclib/fault/detail/fiber/atomic.hpp

```diff
@@ -0,0 +1,328 @@
+#pragma once
+
+#include <yaclib/fault/detail/fiber/atomic_wait.hpp>
+
+namespace yaclib::detail::fiber {
+
+template <typename T>
+class AtomicBase : public AtomicWait<T> {
```

Comment:
Looks ok but where is fault calls here??

Reply:
why? fault will be called in wrapper